src/components/LineIcon.css

```diff
@@ -8,7 +8,7 @@
   color: #00985f;
 }
 
-.TRAIN {
+.RAIL {
```

Comment:
This is used in the line input, so it's not super-relevant to this PR, but I think it's important to unify the terms across the app. RAIL is what is returned from JORE, so that's what we should use.